build_deploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+make osd-container-image-build-push
+
+# Generate version and tag information from inputs
+COMMIT_NUMBER=$(git rev-list `git rev-list --parents HEAD | egrep "^[a-f0-9]{40}$$"`..HEAD --count)
+CURRENT_COMMIT=$(git rev-parse --short=7 HEAD)
+IMAGE_TAG="v0.1.${COMMIT_NUMBER}-${CURRENT_COMMIT}"
+
+docker tag quay.io/app-sre/configuration-anomaly-detection:${IMAGE_TAG} quay.io/app-sre/configuration-anomaly-detection:${CURRENT_COMMIT}
+docker push quay.io/app-sre/configuration-anomaly-detection:${CURRENT_COMMIT}
```

Comment:
you should handle authentication before pushing.

this will include:
- creating a docker config file and performing a login with it. example: https://github.com/app-sre/qontract-server/blob/19a6b118c9b0d9444e405f4c5135bfce50319e09/build_deploy.sh#L4-L6
- pushing while specifying the config file. example: https://github.com/app-sre/qontract-server/blob/19a6b118c9b0d9444e405f4c5135bfce50319e09/Makefile#L61-L63